zellij-client/src/os_input_output.rs

```diff
@@ -88,6 +100,60 @@ pub(crate) fn get_terminal_size_using_fd(fd: RawFd) -> Size {
     Size { rows, cols }
 }
 
+#[cfg(windows)]
+pub(crate) fn get_terminal_size(handle_type: HandleType) -> Size {
+    use std::os::windows::io::RawHandle;
+    use windows_sys::Win32::{
+        Foundation::INVALID_HANDLE_VALUE,
+        System::Console::{
+            GetConsoleScreenBufferInfo, GetStdHandle, CONSOLE_SCREEN_BUFFER_INFO, COORD, SMALL_RECT,
+        },
+    };
+
+    // TODO: handle other handle types, only stdout is supported for now
+    let handle_type = match handle_type {
+        // HandleType::Stdin => windows_sys::Win32::System::Console::STD_INPUT_HANDLE,
+        // HandleType::Stdout => windows_sys::Win32::System::Console::STD_OUTPUT_HANDLE,
+        // HandleType::Stderr => windows_sys::Win32::System::Console::STD_ERROR_HANDLE,
+        _ => windows_sys::Win32::System::Console::STD_OUTPUT_HANDLE,
+    };
```

Comment:
Currently hardcoding this to fetch the terminal size of stdout only, this is because as mentioned in your PR only the stdout is currently working for windows.

We'll want to remove this once stdin works to match the behavior of the unix implementation.